packages/cli/test/unit/commands/redis/keyspace-notifications.unit.test.ts

```diff
@@ -0,0 +1,40 @@
+import {stdout, stderr} from 'stdout-stderr'
+import Cmd from '../../../../src/commands/redis/keyspace-notifications'
+import runCommand from '../../../helpers/runCommand'
+import * as nock from 'nock'
+import {expect} from 'chai'
+import {shouldHandleArgs} from '../../lib/redis/shared'
+
+describe('heroku redis:keyspace-notifications', function () {
+  shouldHandleArgs(Cmd, {config: 'A'})
+})
+
+describe('heroku redis:keyspace-notifications', function () {
+  afterEach(function () {
+    nock.cleanAll()
+  })
+
+  it('# sets the keyspace notify events', async () => {
+    const api = nock('https://api.heroku.com')
+      .get('/apps/example/addons').reply(200, [
+        {name: 'redis-haiku', addon_service: {name: 'heroku-redis'}, config_vars: ['REDIS_FOO', 'REDIS_BAR']},
+      ])
+
+    const redis = nock('https://api.data.heroku.com')
+      .patch('/redis/v0/databases/redis-haiku/config', {notify_keyspace_events: 'AKE'}).reply(200, {
+        notify_keyspace_events: {value: 'AKE', desc: 'Enables keyspace notifications.', default: ''},
+      })
+
+    await runCommand(Cmd, [
+      '--app',
+      'example',
+      '--config',
+      'AKE',
+    ])
+
+    api.done()
```

Comment:
Do we actually need these `.done()` calls?

Reply:
Yes, it's done to ensure that all mocked requests were consumed, that assertion will fail if there's still some mocked request in `nock` that wasn't consumed by the test.